### PR TITLE
Fix pulls creation

### PR DESCRIPTION
**Pull request by justinhyou**

### CODE REVIEW

**Comment by jesswinssinger** [comment]
Weeee merge pls!!!

**Comment by jesswinssinger** [comment]
Hi! 2 lil fixes - 
Error when creating a new pull (it says room_assignments isn't an attribute for room?)
No option to adjust number of students per pull (diff pulls can have diff numbers depending on which rooms they are)

File: app/models/student.rb [hunk 1]
```diff
@@ -39,11 +39,6 @@ def senior?
     class_rank == :senior or class_rank == :super_senior
   end
 
-  def number_sort
```
Comment (g-rocket) [inline app/models/student.rb]: Why are you deleting this?  It's used on the `admin/students` page.

File: config/initializers/urls.rb [hunk 2]
```diff
@@ -1,23 +1,23 @@
-require './lib/google_api/drive.rb'
+# require './lib/google_api/drive.rb'
```
Comment (g-rocket) [inline config/initializers/urls.rb]: Why is this all commented out?  I don't think you meant to push this, but your changes to this file (and `students.rb`) should not be part of this pull request and should not be merged to master.
